src/compounds/ActionsGroup/actions/styles.scss

```diff
@@ -4,3 +4,7 @@
 #send-message {
   height: 150px;
 }
+
+.disclaimer {
```

Comment:
if you didn't want to add a whole custom style for this, you could just use the class of "small" here

Reply:
thanks! I'll take another quick look at it tomorrow. I tried using `fs-6` but it wasn't small enough.

Reply:
additionally it can be inside of `<small></small>` html tags for accessibility since these denote to screen readers that its a disclaimer

Reply:
nice! I added the small tags, but the text was still larger than desired with that alone. so I'm keeping the class and tag for styling plus accessibility. thanks for the note!

ref: [mdn docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/small)